.github/workflows/integration-test-setup/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: Cluster Setup
+description: Cluster setup for integration test
+inputs:
+  kubernetes-version:
+    description: Name to use for the workflow
+    required: true
+  github-token:
+    description: GITHUB_TOKEN from the calling workflow
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+      - name: setup golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: setup minikube
+        uses: manusa/actions-setup-minikube@v2.4.2
+        with:
+          minikube version: "v1.23.2"
+          kubernetes version: ${{ inputs.kubernetes-version }}
+          start args: --memory 6g --cpus=2
+          github token: ${{ inputs.github-token }}
+
+      - name: print k8s cluster status
+        shell: bash --noprofile --norc -eo pipefail -x {0}
```

Comment:
Using a top-level default does not work anymore?

```
defaults:
  run:
    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
    shell: bash --noprofile --norc -eo pipefail -x {0}
```

Reply:
In composite YAML only we have to mention at every place we want to use the shell.